react/components/ProductImages/components/Carousel/styles.css

```diff
@@ -0,0 +1,26 @@
+/**
+*   Carousel CSS Module
+*/
+.carouselGaleryCursor {
+  cursor: url('./cursor.svg') 8 8, default;
+}
+
+.carouselGaleryThumbs :global(.swiper-slide) {
+  height: auto;
+}
+
+.carouselGaleryThumbs :global(.swiper-slide):not(:global(.swiper-slide-thumb-active)) .carouselThumbBorder {
+  border-width: 0px;
```

Comment:
It's possible to use tachyons for this?

Reply:
This one is changing from an external lib, so...